src/components/Slider/BootstrapSlider.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'bootstrap-slider';
+
+class BootstrapSlider extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this._slider = null;
+  }
+
+  componentDidMount() {
+    const that = this;
```

Comment:
@dmiller9911 I am using it because the function is being called in bootstrap-slider's context. 
**this._slider.on('slide', value => that._onSlide(value));**

Reply:
@dmiller9911, 
I moved the `onSlide` function into `componentDidMount`